server/sublist.go

```diff
@@ -266,12 +267,14 @@ func (s *Sublist) Match(subject string) *SublistResult {
 	matchLevel(s.root, tokens, result)
 
 	// Add to our cache
-	s.cache[subject] = result
-	// Bound the number of entries to sublistMaxCache
-	if len(s.cache) > slCacheMax {
-		for k := range s.cache {
-			delete(s.cache, k)
-			break
+	if s.cache != nil {
```

Comment:
I think all code in sublist assumes the cache is not nil since it is (was) created in NewSublist().

Reply:
Ditto